src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/InfoHandler.php

```diff
@@ -171,7 +171,15 @@ public static function getOrderSystemStatus(): array
 
         $orderSystemStatus['last_synchronized_date'] = TaskModel::getLatestSuccessfulSynchronizedDateForType(TaskHandler::ORDERS_EXPORT);
 
-        $orderSystemStatus['ids_with_failed_tasks'] = TaskModel::getFailedOrderIds();
+        $failedOrderIds = TaskModel::getFailedOrderIds();
+
+        $failedOrderIdsWithoutCancelled = array_filter($failedOrderIds, static function (int $orderId) {
+            $order = new WC_Order($orderId);
+
+            return OrderExport::STATUS_CANCELLED !== $order->get_status();
```

Comment:
can we do a single query instead of loading each order?

Reply:
Added WC_Order_Query and filter for cancelled orders